pushtemplates/src/main/java/com/clevertap/pushtemplates/VideoActivity.java

```diff
@@ -151,7 +188,22 @@ public void onClick(View view) {
 					getWindowManager().getDefaultDisplay().getMetrics(metrics);
 					setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
 					FrameLayout.LayoutParams params = (FrameLayout.LayoutParams) aspectRatioFrameLayout.getLayoutParams();
+					Log.d("valueofdensity",String.valueOf( getApplicationContext().getResources().getDisplayMetrics().density));
```

Comment:
@JideshNair And since the same code is repeating everywhere, can you make this a function?

Reply:
fixed